modules/form-field/react/lib/FormField.tsx

```diff
@@ -50,6 +50,8 @@ export interface FormFieldProps
    * The input component wrapped by the FormField.
    */
   children: React.ReactNode;
+  errorLabel?: string;
```

Comment:
Can you document that these props are specifically for translations and should not be used to customize the label? Also defaults in the jsdoc woudl be nice 👍